VMEncryption/main/handle.py

```diff
@@ -686,20 +754,12 @@ def enable_encryption():
         logger.log(msg="azure encryption path creation failed.",
                    level=CommonVariables.ErrorLevel)
 
-    # If the BEK is present try to remount everything
     existing_passphrase_file = bek_util.get_bek_passphrase_file(encryption_config)
-    if existing_passphrase_file is not None:
-        disk_util.consolidate_azure_crypt_mount(existing_passphrase_file)
-        mount_encrypted_disks(disk_util=disk_util,
-                              bek_util=bek_util,
-                              encryption_config=encryption_config,
-                              passphrase_file=existing_passphrase_file)
-    else:
-        if encryption_config.config_file_exists():
-            logger.log(msg="EncryptionConfig is present, but could not get the BEK file.",
+    if existing_passphrase_file is None and encryption_config.config_file_exists():
+        logger.log(msg="EncryptionConfig is present, but could not get the BEK file.",
```

Comment:
I would prefer "could not get the key file" instead of "BEK file" since BEK = Bitlocker Encryption Key and is more of a Windows acronym.

Reply:
Done.